content/en/docs/contribute/review/for-approvers.md

```diff
@@ -222,4 +222,26 @@ https://github.com/kubernetes/kubernetes/issues.
 If this is a documentation issue, please re-open this issue.
 ```
 
+### Squashing
 
+As an approver, when you review pull requests (PRs), there are various cases where you might do the following:
+- Advise the contributor to squash their commits.
+- Squash the commits for the contributor.
+- Advise the contributor not to squash yet.
+- Prevent squashing.
+
+**Advising contributors to squash**: A new contributor might not know that they should squash commits in their pull requests (PRs). If so, advise them to do so, provide links to useful information, and offer to arrange help if they need it. Some useful links:
+-  [Opening pull requests and squashing your commits](/docs/contribute/new-content/open-a-pr#squashing-commits) for documentation contributors.
+- [GitHub Workflow](https://www.k8s.dev/docs/guide/github-workflow/), including diagrams, for developers.
+
+**Squashing commits for contributors**: If a contributor might have difficulty squashing commits or there is time pressure to merge a PR, you can perform the squash for them:
+- The kubernetes/website repo is [configured to allow squashing for pull request merges](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests). (TBD is it?) Simply select the squash commits button.
```

Comment:
I didn't understand the TBD part since this already seems to have been done. Is there some context that I'm missing?